examples/shell_command/shell_command.ui

```diff
@@ -41,9 +41,17 @@
      <property name="text">
       <string>Run Shell Command</string>
      </property>
+     <property name="redirectCommandOutput" stdset="0">
+      <bool>true</bool>
+     </property>
      <property name="allowMultipleExecutions" stdset="0">
       <bool>false</bool>
      </property>
+     <property name="commands" stdset="0">
+      <stringlist>
+       <string>echo &quot;Hello World!&quot;</string>
```

Comment:
Was this left without a command on purpose to test what it does without a command? Should I create a new example or add to this example instead of changing it?

Reply:
This has a command... `echo Hello World!`...

Reply:
Or should have...